nix/status-go/mobile/build.nix

```
@@ -31,7 +31,7 @@ in buildGoPackage {
     ++ optional isAndroid openjdk
     ++ optional isIOS xcodeWrapper;
 
-  ldflags = concatStringsSep " " goBuildLdFlags;
+  ldflags = splitString " " (concatStringsSep " " goBuildLdFlags);
```

Eeeee, shouldn't this just be:
```suggestion
  ldflags = goBuildLdFlags;
```
Seems like you combine it then split it for no reason.

---

Thanks!